t/op/try.t

```diff
@@ -242,6 +242,9 @@ use feature 'try';
     is($scalar, "result", 'do { try/catch } with multiple statements');
 }
 
+my $program = $0;
+$program =~ s/\.dp$//; # running under 'cd t; ./TEST -deparse'
```

Comment:
`$` is semantically wrong; should be `\z`.

Reply:
It makes no difference.